installer-static.tpl.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+# This script is used to install the required packages for the application
+
+# Function for detect OS and set the package manager
+function detect_package_manager() {
+  ARCH=$(uname -m)
```

Comment:
Could you please move it to the `install_zypper` function?

Reply:
Ok